### DISCUSSION TITLE
Hi, how i can change seek buttons to next and prev?

### DISCUSSION THREAD

**Discussion by kazturga** [mediagrid/capacitor-native-audio#7] [1. Help] [answered]
Hi, how i can change seek buttons to next and prev?

**Comment by wsamoht** [comment]
Assuming you are referring to the Now Playing widget on iOS? If so, we only enable seeking within the current audio source. This is because the plugin doesn't have a playlist so there is no next/previous.

**Reply by kazturga** [reply]
Is it possible to make it so that pressing a button simply transmits an event to the application?
For example
next_button_pressed => {}
prev_button_pressed => {}

I only need these events and the ability to turn on the next and previous track buttons, I think I can figure it out with a playlist

**Reply by wsamoht** [reply, marked as answer]
I briefly looked at the code and on iOS handling the button action is already a custom handler so could be modified. On Android, the handler is built into ExoPlayer and overriding that logic I think is possible but not that easy.

For right now, unfortunately this is out of the scope of the functionality we want to offer in the plugin. The best solution would be to implement logic to use the playlist functionality at the native layer.
